starfish/test/spots/detector/test_spot_detection.py

```diff
@@ -25,8 +26,19 @@ def simple_local_max_spot_detector() -> TrackpyLocalMaxPeakFinder:
     )
 
 
+def simple_local_max_spot_detector() -> LocalMaxPeakFinder:
+    return LocalMaxPeakFinder(
+        min_distance=6,
+        stringency=0,
+        min_obj_area=0,
+        max_obj_area=np.inf,
+        threshold=0
```

Comment:
do the tests still pass if threshold=None? i think they should, now that we know that we're no longer running into a "no spots found" error